recipe/bld.bat

```diff
@@ -1,4 +1,11 @@
-move bin\* %LIBRARY_BIN%\
+REM Move only desired files as the downloaded package contains even Qt5 dlls
+move bin\cmake.exe %LIBRARY_BIN%\
+move bin\cmcldeps.exe %LIBRARY_BIN%\
+move bin\cpack.exe %LIBRARY_BIN%\
+move bin\ctest.exe %LIBRARY_BIN%\
+move bin\msvcp120.dll %LIBRARY_BIN%\
```

Comment:
I did not find this package because I was looking for msvc*_runtime. So I'll open another PR to include it as a dependency